casper/src/main/rholang/MakeMint.rho

```diff
@@ -29,7 +29,7 @@ new int2NN, MakeMint, NonNegativeNumber, sgset(`rho:stopgap:set`),
         contract internalMakePurse(@balance, return) = {
           //balance must be a name that NonNegativeNumber contract is listening on
           new thisPurse in {
-            contract thisPurse(=*decr, @amount, success) = { @(balance, "sub")!(amount, *success) } |
+            contract thisPurse(@=*decr, @amount, success) = { @(balance, "sub")!(amount, *success) } |
```

Comment:
Are you taking advantage of the fact that `contract` is syntactic sugar for non-linear `for`s and just put a pattern here?

Reply:
Yes.  The security of the purse relies on a purse of a different currency being unable to construct this first argument.  Which means I have to add a bunch of bundle+ wrappers.  BRB!